### ISSUE TITLE
Clearer colourmap / labels when showing task order in visualize

### ISSUE THREAD

**Issue by TomNicholas** [dask/dask#9069]
After a helpful suggestion from @TomAugspurger, I'm trying to examine the static task order of my DAG. This is harder than it should be just because

1) The current way of applying the colour map means sequential tasks have the most similar colours
2) There is no key to tell me which colour is first
3) I don't know of any other way to access this information
4) This definitely isn't colourblind-friendly (I'm not colourblind, but ~8% of men are)

My looks like this, which I'm finding hard to parse - for example I'm finding hard to tell if 30 occurs before 37.

![image](https://user-images.githubusercontent.com/35968931/167953949-4586ade1-9f57-44b0-9284-ef5deb7657a9.png)

Q's:

1) Is there another way for me to easily find out the static task order?
2) Can we add an (optional) key to help me differentiate?
3) Is there a html version of this graph that can tell me this info with a mouseover?

Thanks

**Comment by TomAugspurger** [comment]
> 1. Is there another way for me to easily find out the static task order?

`dask.order.order` will give you the ordering as a dictionary (I think0.

FYI, `cmap="autumn", node_attr={"penwidth": "4"})` helps a bit with the coloring.

> Is there a html version of this graph that can tell me this info with a mouseover?

I don't think so. That would be handy.

**Comment by ian-r-rose** [comment]
> Is there a html version of this graph that can tell me this info with a mouseover?

This would be a good thing to do as part of #7301. A nicer, more web-friendly graph viz would also be great for #9053 .
